css-layout-classes/styles.css

```diff
@@ -0,0 +1,53 @@
+#container {
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+
+.column-one-third {
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds {
+  flex-basis: 66.6%;
+}
+
+.column-full {
+  flex-basis: calc(100%/1);
+}
+
+.ssbu-card {
+  display: flex;
+  width: 100%;
+  flex-wrap: nowrap;
+}
+
+img {
+  width: 100%;
+}
+
+img.one {
+  width: 250px;
+  height: 250px;
+}
+
+img.two {
+  width: 250px;
+  height: 250px;
+}
+
+img.three {
+  width: 250px;
+  height: 250px;
+}
+
+img.four {
+  width: 250px;
+  height: 250px;
+}
```

Comment:
If we wrap each image with column with correct width then we won't need any of image sizings above